src/templates/dso_api/dynamic_api/api.html

```diff
@@ -63,8 +67,458 @@
         .navbar-brand:hover {
             color: black;
         }
+        .string { color: black; }
+        .bool { color: darkcyan; }
+        .number { color: #195f91 }
+        .null { color: magenta; }
+        .key { 
+          color:  #EC0000; 
+          font-weight: 550;
+        }
+        .link { color: #A30000; }
+        .lit { color: #195f91; }
+        .collapsible {
+            cursor:pointer
+        }
+        .collapsible:not(.collapsed) .collapsible_summary {
+            display: none
+        }
+        .collapsible:hover .collapsible_summary {
+          background: #e4e4e4
+        }
+        .collapsible.collapsed .collapsible_content {
+            display: none
+        }
+        .collapsible:not(.collapsed) .collapsible_content {
+            display: inline-block;
+            vertical-align: top;
+        }
+        nav {
+          float: right;
+          visibility: hidden;
+          height: 56px;
+        }
+        nav.active {
+          visibility: initial;
+        }
+        #page-links .hidden {
+          display: none;
+        }
     </style>
+{% endblock %}
+
+{% block content %}
+
+<div class="region"  aria-label="{% trans "request form" %}">
+{% block request_forms %}
```

Comment:
Als je een blok 1-op-1 overneemt kan je ook dit doen:
```
{% block request_forms %}{{ block.super }}{% endblock %}
```
En anders misschien ff een comment wat er aan gedaan is?

Reply:
Ah dat wist ik niet.